tap_twitter_ads/schema.py

```diff
@@ -103,9 +102,9 @@ def get_schemas(reports):
         # https://github.com/singer-io/singer-python/blob/master/singer/metadata.py#L25-L44
         mdata = metadata.get_standard_metadata(
             schema=schema,
-            key_properties=stream_metadata.get('key_properties', None),
-            valid_replication_keys=stream_metadata.get('replication_keys', None),
-            replication_method=stream_metadata.get('replication_method', None)
+            key_properties= (hasattr(stream_metadata, 'key_properties') or None) and stream_metadata.key_properties,
```

Comment:
What is the benefit of putting None in or condition? If no benefit then removes it.